index.js

```diff
@@ -22,9 +22,53 @@ const removeElements = elements => {
 	}
 };
 
+const scrollToElement = (element, options) => {
+	const hasOverflow = string => /overflow(-x|-y)?: ?( |auto|scroll)+;/.test(string);
+	const findScrollParent = element => {
+		if (element === undefined) {
+			return;
+		}
+
+		if (hasOverflow(element.style.cssText)) {
```

Comment:
It seems a bit fragile to try to match with a regex.

Reply:
I changed the regex with a more robust `isOverflown` function 😉